topk_test.go

```diff
@@ -70,8 +70,8 @@ func TestTopK(t *testing.T) {
 
 	// at least the top 25 must be in order
 	for i := 0; i < 25; i++ {
-		if top[i].Key != freq.keys[i] {
-			t.Errorf("key mismatch: idx=%d top=%s (%d) exact=%s (%d)", i, top[i].Key, top[i].Count, freq.keys[i], freq.counts[freq.keys[i]])
+		if *top[i].Key != freq.keys[i] {
```

Comment:
shouldnt Key be the pointer ?

Reply:
Changed this again, so internally I will be using pointers but when requesting the TopK will convert it not a different struct